src/components/DrawerHeader.vue

```diff
@@ -29,8 +33,7 @@ export default defineComponent({
   },
   emits: ['update:modelValue'],
   setup(props, { emit }) {
-    const dialogOpened = ref(false);
-    const dialogState = ref('default');
+    const isDialogOpen = ref(false);
 
     const classes = computed(() => {
```

Comment:
Missing func input return value type `const classes = computed((): string => {`

Reply:
Solved - removed the remnant of previous drawer solution: this is no logner used (replaced by bottom panel).